### ISSUE TITLE
Restart-Computer and Stop-Computer fails

### ISSUE THREAD

**Issue by NJ-Dude** [PowerShell/PowerShell#7798]
Steps to reproduce
------------------

```powershell
- Open Powershell Core
- Type: Restart-Computer
```

Expected behavior
-----------------

```none
- The system will reboot

(Same results if Stop-Computer is used, the system in this case
won't shut down but display a similar error message)
```

Actual behavior
---------------

```none
The following error occurs:

PS C:\Users\Developer> Restart-Computer
Restart-Computer : Failed to restart the computer Development-PC with the following error message: The client cannot connect to the destination specified in the request. Verify that the service on the destination is running and is accepting requests. Consult the logs and documentation for the WS-Management service running on the destination, most commonly IIS or WinRM. If the destination is the WinRM service, run the following command on the destination to analyze and configure the WinRM service: "winrm quickconfig"..
At line:1 char:1
+ Restart-Computer
+ ~~~~~~~~~~~~~~~~
+ CategoryInfo          : OperationStopped: (Development-PC:String) [Restart-Computer], InvalidOperationException
+ FullyQualifiedErrorId : RestartcomputerFailed,Microsoft.PowerShell.Commands.RestartComputerCommand

```

Environment data
----------------

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      6.1.0
PSEdition                      Core
GitCommitId                    6.1.0
OS                             Microsoft Windows 10.0.17134
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```

**Comment by chucklu** [comment]
@NJ-Dude  just run command as the error message told you` winrm quickconfig`

I also suggest you enable all options when you install the powershell, the third option related the problem you encountered.
![2018-09-18_10-36-40](https://user-images.githubusercontent.com/3004057/45728078-0a5e4080-bbf9-11e8-8b32-c4207ddad40a.png)

**Comment by alexandair** [comment]
@chucklu Why is remoting needed to restart the localhost?

**Comment by chucklu** [comment]
@alexandair  Actually I am not sure about this, I just guess it according my experience.
When I tick the third option, then I do not need to run the `winrm quickconfig`, and the stop-computer works. Otherwise I need do it manually.

You can check my blog to view the procedure of `winrm quickconfig`
https://www.cnblogs.com/chucklu/p/7491868.html
In step4, you need enable the remote access

> WinRM is not set up to **allow remote access to this machine for management.**
> The following changes must be made:
> 
> Create a WinRM listener on HTTP://* to accept WS-Man requests to any IP on this machine.
> Enable the WinRM firewall exception.
> 
> Make these changes [y/n]? y

**Comment by SteveL-MSFT** [comment]
I took a look at the [code](https://github.com/PowerShell/PowerShell/blob/5d03e1653a7d518715fa3f00587cad6b5c78cc89/src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs#L864) and it's relyling on calling WMI via WinRM to invoke the shutdown (even in local case).

Looking at the Windows PowerShell code base, it's able to call into WMI using DCOM instead of via WinRM.  This code path is no longer available in PowerShell Core as .NET Core doesn't support DCOM.

TL;DR; `*-Computer` cmdlets require WinRM even locally.

**Comment by chucklu** [comment]
@NJ-Dude  make sure you are open the powershell as Administrator

**Comment by NJ-Dude** [comment]
I did open as Administrator, same results if you are admin or not

**Comment by chucklu** [comment]
@NJ-Dude  I am not sure why it did not work on your computer.
Anyway, you can check my blog to view the procedure of winrm quickconfig
https://www.cnblogs.com/chucklu/p/7491868.html
